index.html

```diff
@@ -228,31 +258,26 @@ <h3>
         </li>
         <li>
           <a>For each</a> <var>entry</var> of <var>entries</var>:
-        </li>
-        <li>
           <ol>
-            <li>If <var>entry</var>["src"] is not <code>undefined</code>:
-              <ol>
-                <li>Let <var>image</var> be a new object created as if by the
-                expression ({}).
-                </li>
-                <li>Set <var>image</var>["src"] to the result of <a>parsing</a>
-                <var>entry</var>["src"] using <var>context</var> as the
-                <var>base URL</var>.
-                </li>
-                <li>Set <var>image</var>["type"] to the result of running
-                <a>processing the <code>type</code> member of an image</a>
-                given <var>entry</var>.
-                </li>
-                <li>Set <var>image</var>["sizes"] to the result of running <a>
-                  processing the <code>sizes</code> member of an image</a>
-                  given <var>entry</var>.
-                </li>
-                <li>
-                  <a data-lt="list-append">Append</a> <var>image</var> to
-                  <var>imageResources</var>
-                </li>
-              </ol>
+            <li>Let <var>image</var> be a new <a>processed image resource</a>.
+            </li>
+            <li>Set <var>image</var>["src"] to the result of <a>parsing</a>
```

Comment:
Since this is a spec-level object, not a JS object, don't "index" its properties using JS syntax. Refer to them like this:

"Set *image*'s src to the result of parsing *entry*["src"]" (since *entry* is a JS object but *image* is a conceptual/spec object).